sechub-cli/src/mercedes-benz.com/sechub/cli/config_test.go

```diff
@@ -133,3 +133,30 @@ func Example_validateRequestedReportFormatMakesLowercase2() {
 	// NOTICE: Converted requested report format 'Json' to lowercase. Because it contained uppercase characters, which are not accepted by SecHub server.
 	// json
 }
+
+func Example_normalizeCMDLineArgs() {
```

Comment:
You are testing only the happy path. How about the following cases:

~~~
argListXY := []string{"./sechub", "-jobUUID",  "getReport", "my-sechub.json", "-configfile", "3bdcc5c5-c2b6-4599-be84-f74380680808"}
argListYX := []string{"./sechub", "-jobUUID",  "my-sechub.json", "-configfile", "3bdcc5c5-c2b6-4599-be84-f74380680808", "getReport"}
argListXX := []string{"./sechub", "getReport", "-jobUUID",  "my-sechub.json", "-configfile", "3bdcc5c5-c2b6-4599-be84-f74380680808"}
~~~

Reply:
Okay.